docs/release-notes/v1.25.X.md

```diff
@@ -579,4 +580,4 @@ Before upgrading from earlier releases, be sure to read the Kubernetes [Urgent U
 * Fix deprecation message [(#6112)](https://github.com/k3s-io/k3s/pull/6112)
 * Added warning message for flannel backend additional options deprecation [(#6111)](https://github.com/k3s-io/k3s/pull/6111)
 
------
+-----

```

Comment:
I don't know why github diff went crazy on the v1.25 notes, all that was added is the single sidebar line